src/txmempool.h

```diff
@@ -78,7 +80,8 @@ class CTxMemPoolEntry
 
 public:
     CTxMemPoolEntry(const CTransaction& _tx, const CAmount& _nFee,
-                    int64_t _nTime, double _dPriority, unsigned int _nHeight, bool poolHasNoInputsOf = false);
+                    int64_t _nTime, double _dPriority, unsigned int _nHeight, bool poolHasNoInputsOf = false,
+                    unsigned int nSigOps = 0);
```

Comment:
It does not make sense to default it period. Note the exact sigop count is part of the GBT response.